chaste_codegen/data/tests/cellml/hilgemann_noble_model_1987.cellml

```diff
@@ -75,16 +75,10 @@ http://www.CellML.org/
       <unit units="micrometre3" exponent="-1"/>
    </units>
    <component name="environment">
-      <variable name="time" public_interface="out" units="second" cmeta:id="time">
-         <rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:bqbiol="http://biomodels.net/biology-qualifiers/">
-            <rdf:Description rdf:about="#time">
-               <bqbiol:is rdf:resource="https://chaste.comlab.ox.ac.uk/cellml/ns/oxford-metadata#time"/>
-            </rdf:Description>
-         </rdf:RDF>
-      </variable>
+      <variable name="time" units="second" public_interface="out"/>

```

Comment:
Why have all the CellML files changed too?

Reply:
because I've reverted to the original cellml files just with metadata tags added.
What I had done initially is just get the new versions, which was fine for normal conversions but PyCml didn't generate jacobians for a number of the new cellml files, so I reverted in order to compare apples with apples.